src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.State.cs

```diff
@@ -408,6 +400,23 @@ private State()
                     lastAdditionalDependentFileTimesChangedAtUtc: lastAdditionalDependentFileTimesChangedAtUtc,
                     lastItemsChangedAtUtc: lastItemsChangedAtUtc, lastCheckedAtUtc: LastCheckedAtUtc);
 
+
+                DateTime getLastTimeAdditionalDependentFilesAddedOrRemoved(IImmutableDictionary<string, DateTime> additionalDependentFileTimes)
```

Comment:
```suggestion
                DateTime GetLastTimeAdditionalDependentFilesAddedOrRemoved()
```

Reply:
> 

additionalDependentFileTimes is passed to create a new State(..)

Reply:
done in next commit